core_backend/app/dashboard/models.py

```diff
@@ -629,6 +632,177 @@ def convert_rows_to_top_content_time_series(
+            and_(
+                func.date_trunc(interval_str, ContentFeedbackDB.feedback_datetime_utc)
+                == query_count_ts.c.time_period,
+                ContentFeedbackDB.content_id == content_id,
+                ContentFeedbackDB.user_id == user_id,
+            ),
+            isouter=True,
+        )
+        .group_by(
+            query_count_ts.c.time_period,
+            query_count_ts.c.content_title,
+            query_count_ts.c.query_count,
+        )
+        .order_by(query_count_ts.c.time_period)
+    )
+
+    user_feedback = (
+        select(
+            ContentFeedbackDB.feedback_datetime_utc,
+            QueryDB.query_text,
+            ContentFeedbackDB.feedback_text,
+        )
+        .join(QueryDB)
+        .where(
+            ContentFeedbackDB.content_id == content_id,
+            ContentFeedbackDB.user_id == user_id,
+            ContentFeedbackDB.feedback_datetime_utc >= start_date,
+            ContentFeedbackDB.feedback_datetime_utc < end_date,
+            ContentFeedbackDB.feedback_text.is_not(None),
+        )
+        .order_by(ContentFeedbackDB.feedback_datetime_utc.desc())
+        .limit(10)
+    )
+
+    result_ts = await asession.execute(feedback_ts)
+    result_feedback = await asession.execute(user_feedback)
+
+    rows_ts = result_ts.fetchall()
+    rows_feedback = result_feedback.fetchall()
+
+    format_str = "%Y-%m-%dT%H:%M:%S.000000Z"  # ISO 8601 format (required by frontend)
+    return convert_rows_to_details_drawer(
+        timeseries=rows_ts,
+        feedback=rows_feedback,
+        format_str=format_str,
+        n_days=day_between,
+    )
+
+
+def convert_rows_to_details_drawer(
+    timeseries: Sequence[Row[Any]],
+    feedback: Sequence[Row[Any]],
+    format_str: str,
+    n_days: int,
+) -> DetailsDrawer:
+    """
+    Convert rows to DetailsDrawer
+    """
+    time_series = {}
+    query_count = 0
+    positive_count = 0
+    negative_count = 0
+    title = ""
+
+    if timeseries:
+        title = timeseries[0].content_title
+
+    for r in timeseries:
+        time_series[r.time_period.strftime(format_str)] = {
+            "query_count": r.query_count,
+            "positive_count": r.positive_count,
+            "negative_count": r.negative_count,
+        }
+        query_count += r.query_count
+        positive_count += r.positive_count
+        negative_count += r.negative_count
+
+    feedback_rows = []
+    for r in feedback:
+        feedback_rows.append(
+            UserFeedback(
+                timestamp=r.feedback_datetime_utc.strftime(format_str),
+                question=r.query_text,
+                feedback=r.feedback_text,
+            )
+        )
+
+    return DetailsDrawer(
+        title=title,
+        query_count=query_count,
+        positive_votes=positive_count,
+        negative_votes=negative_count,
+        daily_query_count_avg=query_count // n_days,
+        time_series=time_series,
+        ai_summary="Not Available",
```

Comment:
Hard coded for now. 
In the future, if env var is set, it will generate AI Summary else it will just return "Not Available".